framework/vst/vstmodule.cpp

```diff
@@ -0,0 +1,77 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#include "vstmodule.h"
+#include "settings.h"
+
+#include "vstscaner.h"
+#include "log.h"
+
+using namespace mu::vst;
+using namespace mu::framework;
+
+VSTConfiguration VSTModule::m_configuration = VSTConfiguration();
+
+std::string VSTModule::moduleName() const
+{
+    return "vst";
+}
+void VSTModule::registerExports()
+{
+
+}
+void VSTModule::resolveImports()
+{
+
+}
+
+void VSTModule::onInit()
+{
+    m_configuration.init();
+    VSTScaner scaner(m_configuration.searchPaths());
```

Comment:
It is better to move this logic to some service. 
Do we need to load plugins on a different thread?

Reply:
As said in PR's comment:
> Method VSTModule::onInit() temporary includes example of using.